README.md

```diff
@@ -310,6 +319,50 @@ var numbers = arrayType.children(function(el) {
 
 Because only children are tested with the condition function, the values `[1, 2]` are seen as an `array` type whose content is never tested. Thus, the only direct child which is a number type is `3`.
 
+##### getById
+
+Search the entire tree to find a matching ID.
+
+```javascript
+elTree.getById('some-id');
+```
+
+##### contains
+
+Test to see if a collection contains the value given.
```

Comment:
Should elaborate on whether this is done via deep equality. E.g., does `arrayType.contains([1, 2])` return `true` or `false`?